creator-node/sequelize/migrations/20200911004845-allow-track-and-audiusUsers-appends.js

```diff
@@ -0,0 +1,64 @@
+'use strict'
+
+module.exports = {
```

Comment:
just documenting here that as a whole migration accomplishes: replace Files.trackUUID with Files.blockchainId, drop Tracks.trackUUId, drop AudiusUsers.audiusUserUUID